src/pattern-library/util/jsx-to-string.js

```diff
@@ -66,8 +66,16 @@ export function jsxToString(vnode) {
           return value ? name : '';
         }
 
-        const valueStr =
-          typeof value === 'string' ? `"${escapeQuotes(value)}"` : `{${value}}`;
+        let valueStr;
```

Comment:
Ran into some challenges with `Symbol` props and `vNode`/component props when spiking.